app/templates/gentelella/users/events/_table.html

```diff
@@ -1,6 +1,23 @@
 {% macro events_table(events) %}
-    <div class="table-responsive">
-        <table class="table table-striped">
+
+<script src="{{ url_for('static', filename='vendor/datatables.net/js/jquery.dataTables.min.js') }}"></script>
+<script src="{{ url_for('static', filename='vendor/datatables.net-bs/js/dataTables.bootstrap.min.js') }}"></script>
+<script type = "text/javascript" >
+	 $('#dodo').DataTable({
```

Comment:
See line 2 of `app/templates/gentelella/users/events/index.html` ... it includes the file `app/templates/gentelella/users/events/_table.html` . And scripts should be placed inside the `tail_js` block. Not in the middle of a template. 

Please try to understand how Flask templates work and efficient JavaScript usage.

Flask templates: http://flask.pocoo.org/docs/0.12/tutorial/templates/
JS: https://www.w3schools.com/js/js_performance.asp